### PR TITLE
Feature branch

### PR DESCRIPTION
**Pull request by S-BLH**
1. Created a repository by the name of tribute-page
2. Cloned it to my local machine
3. Used github workflow
4. Setup the linters
5. then added the necessary codes for html and css to create the project

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>freeCodeCamp Survey Form</title>
+  <link rel="stylesheet" href="styles.css">
```
Comment (ahzamir) [inline index.html:7]: You have a typo issue in your `styles.css ` which should be `style.css`.

File: style.css
Comment (ahzamir) [inline style.css:1]: [OPTIONAL] - Add some design to your elements and your main page and make it look more awesome so you can share a great view of it using a screenshot.

**Review by ahzamir — COMMENTED** [review]
### Hi @S-BLH 🙋‍♀️,

> Excellent work! 💯 You have done a great job implementing all the project requirements. 🥳 Your feature is complete! Congratulations!!!🎊

- Although your code passes all checks in the freecodecamp and you implemented all of the requirements. I want you to make some changes and add some design to your elements to make it look cool so you can share your project on social media.
- **[REQUIRED]** Share your project on this [GROUP](https://www.facebook.com/groups/kick-startdev), add a simple Proud Moment text and a screenshot of your project then besides that add your project link at the end of the text as a GitHub Source code reference and post.
- Share your Project on your LinkedIn account and add a proud moment text on it with a screenshot.

# Status: Approved ✅
There is nothing to say other than to merge your branch. 🎉🎯✅👍😊🥳

GreatJobThumbsUpGIF

 ## Highlights 🌞

🎉 Great Design Implemented 
✔️ Created a professional Repo (You created a repository with your project name)
✔️ Correct use of Gitflow (You created the development branch and made it your default branch, then created and committed your work on your feature branch)
✔️ Commit messages are meaningful
✔️ Setup the linters correctly (You added the correct linters config into your project according to the language you used on your code)
✔️ No linter errors (All linters checks have been passed 🟢)
✔️ You have followed the Odin project task and committed using the command line
✔️ Created the pull request (From your feature branch to the development branch you created a pull request and submitted the pull request link for review)
✔️ Good PR description (You added a meaningful description to your PR about your work and the features you added on this pull request)

**Cheers 🥂 and Happy coding!!! 👯 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me @ahzamir in your question so I can receive the notification.**

File: index.html (hunk 1 above)
Comment (S-BLH) [reply index.html:7]: Salam Ostad!
Thank you for your amazing feedbacks on my 3rd project. I modified the typo.

File: style.css
Comment (S-BLH) [reply style.css:1]: Insha Allah in next project I will work on the design too.

Comment (ahzamir) [reply style.css:1]: @S-BLH 
- Not in the next project give this project some design so you can share it with others!

Comment (S-BLH) [reply style.css:1]: I applied some designs sir can you review it?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
```
Comment (ahzamir) [inline index.html:7]: - Still, you have a typo issue here and you have to solve it.

**Review by ahzamir — COMMENTED** [review]
@S-BLH 
- Your project is approved but you have to add a little bit of design to your project and also this project is about a tribute page so add some content about someone and make it a tribute page:
  - What now I can see is that you added an image which is not perfect and also a title survey form and a small text.
  - I accept that your project meets the requirements but you need to work it to make it better.

**Pull request merged**